tools/infer/predict_system.py

```diff
@@ -65,15 +65,21 @@ def draw_crop_rec_res(self, output_dir, img_crop_list, rec_res):
         self.crop_image_res_index += bbox_num
 
     def __call__(self, img, cls=True):
+        if img is None:
```

Comment:
The assertation here prevent img from being None:https://github.com/PaddlePaddle/PaddleOCR/blob/fac03876f39bc67acc8eef6d7facb9a2206eeecd/paddleocr.py#L522

Reply:
Fixed.

Reply:
@shiyutang: this is actually incorrect as, let's say `img` is `str` so it's not `None` and assertion is passed there, but then we load it in `check_img` and if it's broken then the method returns `None`. So `img` is `None` in that case.

Reply:
You are right, especially when img is provided in list, it is not checked.